src/Plane.js

```diff
@@ -0,0 +1,24 @@
+class Plane {
```

Comment:
GOOD 2: This class contains the bare minimum for what it needs to do, falling towards single responsability. Variable names are easy to understand.
BAD 2: Whilst the names are easy to understand, maybe a comment on something like the airportId would be helpful to let a maintainer know where the airport id is used in the greater context.
GOALS: Keep classes as simple, functional and as close to single responsability as possible.